spec/controllers/ansible_playbook_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe AnsiblePlaybookController do
+
+  include CompressedIds
+
+  before do
+    EvmSpecHelper.create_guid_miq_server_zone
+    login_as FactoryGirl.create(:user_admin)
+  end
+
+  context "#show" do
+    let(:playbook) { FactoryGirl.create(:embedded_playbook) }
+    subject { get :show, :params => {:id => playbook.id} }
+    render_views
+
+    it "returns status 200" do
+      is_expected.to have_http_status 200
+    end
+
+    it "renders correct template" do
+      is_expected.to render_template(:partial => "layouts/_textual_groups_generic")
+    end
+  end
+
+  context "#show_list" do
+    subject { get :show_list }
+    render_views
+
+    it "returns status 200" do
+      is_expected.to have_http_status 200
+    end
+
+    it "renders correct template" do
+      is_expected.to render_template(:layout => "layouts/application")
```

Comment:
Thank you for your suggestion :) 
But after I made change I ran rspec on that test to be sure that everything is working properly and it is now failing because ```layouts/application``` is being rendered instead of ```layouts/_gtl```. Is this intended behaviour? Should I push it anyway? Here is output of rspec: 
```
  1) AnsiblePlaybookController#show_list renders correct template
     Failure/Error: is_expected.to render_template(:layout => "layouts/_gtl")
     
       expecting layout <layouts/_gtl> but action rendered <["layouts/application", "application"]>.
       Expected ["layouts/application", "application"] to include "layouts/_gtl".
     # ./spec/controllers/ansible_playbook_controller_spec.rb:32:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:75:in `block (3 levels) in <top (required)>'
     # /home/martin/Documents/manageIQ/manageiq/spec/support/evm_spec_helper.rb:28:in `clear_caches'
     # ./spec/spec_helper.rb:75:in `block (2 levels) in <top (required)>'
```